roles/tile.rb

```diff
@@ -37,16 +37,17 @@
     :settings => {
       :defaults => {
         :max_connections => "250",
-        :temp_buffers => "32MB",
+        :shared_buffers => "16GB",
```

Comment:
This is currently configured on a per-server basis, though I suspect some may have been copy and pasted without paying attention to the amount of memory in a new server.

Reply:
I'd like to have a sensible default. Do you think I should remove the other settings of it where it's close to this?

Reply:
I think it was originally done as a proportion of memory although I as say I'm not sure it hasn't sometimes been copy and pasted instead of recalculated for new servers.

Probably machines these days always have sufficiently vast amounts of memory that we can just use a single value.